src/Type/Introspection.php

```diff
@@ -184,6 +185,34 @@ public static function getTypes()
         ];
     }
 
+    /**
+     * Build an introspection query from a Schema
+     *
+     * Introspection is useful for utilities that care about type and field
+     * relationships, but do not need to traverse through those relationships.
+     *
+     * This is the inverse of BuildClientSchema::build(). The primary use case is outside
+     * of the server context, for instance when doing schema comparisons.
+     *
+     * Options:
+     *   - descriptions
+     *     Whether to include descriptions in the introspection result.
+     *     Default: true
+     *
+     * @param array<string, bool> $options
+     *
+     * @return array<string, mixed[]>|null
```

Comment:
I'd unify the style to document array (eg. PHPStorm complains when mixing them)

```suggestion
     * @return array<string, array<mixed>>|null
```